typescript/src/lib/bitcoin/address.ts

```diff
@@ -54,35 +53,33 @@ function publicKeyHashToAddress(
  * provided address is not PKH-based.
  * @param address P2PKH or P2WPKH address that will be decoded.
  * @param bitcoinNetwork Network the address should be decoded for.
- * @returns Public key hash decoded from the address. This will be an unprefixed
- *        hex string (without 0x prefix).
+ * @returns Public key hash decoded from the address.
  */
 function addressToPublicKeyHash(
   address: string,
   bitcoinNetwork: BitcoinNetwork
-): string {
+): Hex {
   const network = toBitcoinJsLibNetwork(bitcoinNetwork)
 
   try {
     // Try extracting hash from P2PKH address.
-    const hash = payments.p2pkh({ address: address, network }).hash!
-    return hash.toString("hex")
+    return Hex.from(payments.p2pkh({ address: address, network }).hash!)
   } catch (err) {}
 
   try {
     // Try extracting hash from P2WPKH address.
-    const hash = payments.p2wpkh({ address: address, network }).hash!
-    return hash.toString("hex")
+    return Hex.from(payments.p2wpkh({ address: address, network }).hash!)
   } catch (err) {}
 
+  // If neither of them succeeded, throw an error.
   throw new Error("Address must be P2PKH or P2WPKH valid for given network")
 }
 
 /**
  * Converts an address to the respective output script.
  * @param address BTC address.
  * @param bitcoinNetwork Bitcoin network corresponding to the address.
- * @returns The un-prefixed and not prepended with length output script.
+ * @returns The output script not prepended with length.
```

Comment:
We can update the docstring of `outputScriptToAddress` in the same way.

Reply:
Done in 794b32c8871c0ee064c2b05b49c6f783fa405da4.